canonical_sphinx/config.py

```diff
@@ -175,10 +175,14 @@ def config_inited(_app: Sphinx, config: Any) -> None:  # noqa: ANN401
         ("github_issues", "enabled"),
         ("discourse", "https://discourse.ubuntu.com"),
         ("sequential_nav", "none"),
-        ("display_contributors", False),
+        ("display_contributors", True),
     ]
 
+    has_contributor_listing = "canonical.contributor-listing" in _app.extensions
```

Comment:
now that we're using `_app` we should rename it to `app`

Reply:
I just hijacked what was already in place in the config_inited function (`def config_inited(_app: Sphinx, config: Any) -> None:  # noqa: ANN401`) I'm happy to change it but I'm not really across all the best practices.

I'm happy to change things if you have suggestions (and I'm always happy to learn), but you may have to be specific.

Reply:
Ah, I see. There's a linter that complains if a function/method parameter is not used. The convention (that the linter follows) is that parameters that aren't used on purpose are prefixed with underscore.
Now that the parameter is used, we can remove the underscore and the linter will be happy

Reply:
That makes sense. I've submitted a fix.